### ISSUE TITLE
Terms and conditions functionality

### ISSUE THREAD

**Issue by steelanterns** [laravel/framework#38771]
- Laravel Version: 8.55.0

- PHP Version: ^7.3|^8.0
- Database Driver & Version:

### Description:
since i run composer update from 8.41.0 to  8.55.0 laravel version
i have this error :
Class 'League\CommonMark\Environment' not found
it worked well before...

**Comment by Jubeki** [comment]
You probably used CommonMark v1.6 before and now you use v2.0
Laravel supports both so the error should be in your Code base.

You can either fix the CommonMark Version v1.6 in the composer.json for now with
```bash
composer require league/commonmark:^1.6
```

Or update the used reference to use the newer version instead. (Please take a look at the upgrade guide of CommonMark for further information)

Namespace of the new Environment Class:
```php
use League\CommonMark\Environment\Environment;
```

**Comment by steelanterns** [comment]
> You probably used CommonMark v1.6 before and now you use v2.0
> Laravel supports both so the error should be in your Code base.
> 
> You can either fix the CommonMark Version v1.6 in the composer.json for now with
> 
> ```shell
> composer require league/commonmark:^1.6
> ```
> 
> Or update the used reference to use the newer version instead. (Please take a look at the upgrade guide of CommonMark for further information)
> 
> Namespace of the new Environment Class:
> 
> ```
> use League\CommonMark\Environment\Environment;
> ```

thanks for your help it works
